buildkitd/buildkitd.go

```diff
@@ -44,35 +44,35 @@ var TCPAddress = "tcp://127.0.0.1:8372"
 
 // TODO: Implement all this properly with the docker client.
 
-// NewClient returns a new buildkitd client.
-func NewClient(ctx context.Context, console conslogging.ConsoleLogger, image string, settings Settings, opts ...client.ClientOpt) (*client.Client, error) {
+// NewClient returns a new buildkitd client, together with a boolean specifying whether the buildkit is local.
```

Comment:
Could we just make `isLocal` public and call it where needed above? Since all but one invocation ignores that parameter.

Reply:
Would just make this somewhat cleaner?

Reply:
Done.